src/panels/ShareModelPanel/ShareModel.tsx

```diff
@@ -99,27 +99,42 @@ export default function ShareModel() {
     );
   };
 
+  const updatePermissions = async (
+    action: string,
+    user: string,
+    updatedAccess: any,
```

Comment:
Can you add a proper type here. I'm surprised the linter didn't complain about an `any`....